src/plugins/git/pluginAdapter.js

```diff
@@ -79,4 +55,36 @@ class PluginAdapter implements IPluginAdapter {
       },
     ];
   }
+  async load(
+    repoOwner: string,
+    repoName: string
+  ): Promise<IDynamicPluginAdapter> {
```

Comment:
A client who has specifically a Git plugin static adapter and calls
`load` might hope to receive (a promise for) specifically a Git plugin
dynamic adapter, perhaps to call extra methods on it, like, “who
authored this commit?”. With the current typing, the promise resolves
only to an unspecified plugin dynamic adapter. No change requested; we
can refine this if necessary/useful.